benchmarks/test_massdef.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
These look like unit tests. Those now go in the main code. The basic distinction is whether or not the test reads data from disk. If it does, it is a benhcmark, otherwise it is a unit test.

Reply:
I think only the first one was a unit test (the other ones are actually all reading from disk). I've moved this one to `pyccl/tests/test_massdef.py`

Reply:
great!